src/message/bulkMessage.js

```diff
@@ -151,6 +151,25 @@ class BulkMessage{
         this.apiTemplate = value;
     }
 
+    /**
+     * Sets the ampBody for the message.
+     * (Optional) Either textBody or HtmlBody must be used or use a ApiTemplate with an AmpBody
```

Comment:
rephrase this line: "Either TextBody or HtmlBody must be used with the AmpBody or use a ApiTemplate"